test-unit.sh

```diff
@@ -22,7 +20,7 @@ unit_test() {
   service=$1
   pushd services/$service
   install_deps
-  serverless testUnit --stage $stage
+  yarn run coverage
```

Comment:
This is a unit test fix, covered by another ticket, and I believe it will be merged prior to this PR.  So I won't bother doc'ing it here until needed.